TestFoundation/TestBundle.swift

```diff
@@ -27,6 +27,21 @@ internal func testBundleName() -> String {
     return testBundle().infoDictionary!["CFBundleName"] as! String
 }
 
+internal func xdgTestHelperURL() -> URL {
+#if DARWIN_COMPATIBILITY_TESTS
+    let exeName = "/xdgTestHelper"
+#elseif os(macOS)
+    let exeName = "/xdgTestHelper.app/Contents/MacOS/xdgTestHelper"
+#else
+    let exeName = "/xdgTestHelper/xdgTestHelper"
+#endif
```

Comment:
This seems ok, but it's somewhat ironic that bundle can't find this helper itself. That's what it's supposed to be there fore in the first place.

Reply:
The issue here is that we are not building xdgTestHelper in a bundle directory structure. It may be worth it to figure that out.

Reply:
I'll aim to do that in a follow-up PR as it is complicated by the fact that there are 3 separate targets being built (Linux, macOS and the Darwin Compatibility testing). However it should act as a good test of the `Bundle` class so it is definitely worth doing.